kod/object/active/holder/room.kod

```diff
@@ -3957,5 +3958,19 @@ messages:
       return;
    }
 
+   DoorLockedSound(what=$)
```

Comment:
Name for this function could be better, as could its documentation.  "DoorLockedSound" makes it sound like it returns the appropriate sound resource, whereas it actually assumes that "what" is a player (rename?), and sends the sound to the client.

Reply:
Yes, absolutely. A slip-up on my part. I've made the updates, and it's ready for another review.